vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -0,0 +1,127 @@
+# Troubleshoot RNW dependencies
```

Comment:
Should this be run as part of the CI to verify that it runs, and if there are any requirements that we are manually installing as part of CI, hopefully this could be used to install them?

Reply:
Yes, but I wanted to get this in first to make sure the requirement for inner loop for devs is explicit but we could augment the CI with this. Note however that CI might have dependencies installed in the image, not through choco which is what we recommend for individuals to use.